scripts/linters/general_purpose_linter.py

```diff
@@ -243,6 +243,18 @@
             'if you\'re having trouble declaring types.'),
         'excluded_files': (),
         'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'\$broadcast'),
+        'message': (
+            'Please do not use $broadcast/$on for propagating events. '
+            'Use EventEmitters and Subscriptions instead.'),
```

Comment:
@srijanreddy98, Are we expecting to use Subscriptions for events? IIRC correctly we had thought of doing it through from parent to child node through input, right?

Reply:
Changed